packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -398,11 +386,17 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
     }
 
     return (
-      <div ref={this._root} {...divProps} role={pageElements.length > 0 ? role : undefined} className={css('ms-List', className)}>
-        <div ref={this._surface} className={'ms-List-surface'} role="presentation">
-          {pageElements}
+      <>
+        <div ref={this._root} {...divProps} role={pageElements.length > 0 ? role : undefined} className={css('ms-List', className)}>
+          <div ref={this._surface} className={'ms-List-surface'} role="presentation">
+            {pageElements}
+          </div>
         </div>
-      </div>
+        <EventListener target={window} type="resize" listener={this._onAsyncResize} />
+        <EventListener capture targetRef={this._root} type="focus" listener={this._onFocus} />
+        <EventListener capture targetRef={this._root} type="scroll" listener={this._onScroll} />
+        <EventListener capture targetRef={this._root} type="scroll" listener={this._onAsyncScroll} />
```

Comment:
`this._root` is not equivalent to `this._scrollElement` which is returned via `findScrollableParent` call. This seems like a change in behavior that may break folks cc: @ThomasMichon

Reply:
I've changed this - had to use target instead of `targetRef` because its type isn't the same as `this._scrollElement`